packages/@aws-cdk/aws-config/private/util.ts

```diff
@@ -0,0 +1,14 @@
+import { CfnConfigRule, Scope } from '../lib';
+
+/**
+ * renders the CloudFormation scope property configuration for AWS Config rule
+ * @param scope The resources that will be tracked by AWS Config
+ */
+export function renderScope(scope?: Scope): CfnConfigRule.ScopeProperty | undefined {
```

Comment:
This feels like premature refactoring. It's not being tested directly or used by multiple other files, and it's only a 7-line method. Not clear why it deserves its own file.

Reply:
sure, i'll plug them back in directly in the 2 usage sites. I'm not strongly opposed to it being duplicated locally within a file.

Reply:
I'm happy for it to be a function that both usage sites can use; I wasn't as clear as to why it was in its own file.